resources/rust/main.rs

```diff
@@ -0,0 +1,17 @@
+// Import the interface to Tessel hardware
+extern crate rust_tessel as tessel;
+
+fn main() {
+  
+  // Set the led pins as outputs with initial states
+  let led1 = tessel.led[0].output(1);
```

Comment:
The example code doesn't match the API of the rust-tessel lib, and also the API couldn't actually work quite like that.

It uses `tessel` as if there were a variable named `tessel` holding a struct with an `led` field, but `extern crate rust_tessel as tessel;` only brings in a `tessel` module. Modules and variables are in separate namespaces. Unlike in JS, Rust modules are just collections of function/structure/trait/constant/etc definitions, and don't contain object instances (global variables are possible, but global mutable state is highly discouraged).

You could do:

```
let tessel = tessel::new();
tessel.led[0].output(1);
```

or 

```
tessel::led(0).output(1)
```

(assuming that the library implemented that)